scripts/griddle.jsx

```diff
@@ -649,7 +650,7 @@ var Griddle = React.createClass({
         ) : "";
     },
     getCustomGridSection: function(){
-        return <this.props.customGridComponent data={this.props.results} className={this.props.customGridComponentClassName} />
+        return <this.props.customGridComponent data={this.props.results} className={this.props.customGridComponentClassName} {... this.props.gridMetadata} />
```

Comment:
Should this be spread operator? I think you could get away with just using `this.props.gridMetadata`. Also it looks like it has a space before `this.props.gridMetadata`.